ctl/src/command.rs

```diff
@@ -52,17 +53,19 @@ macro_rules! command_timeout {
 
       thread::spawn(move || {
         $block
-        send.send(()).unwrap();
+        serde::__private::Ok::<(), anyhow::Error>(send.send(())?)
+        
       });
 
       if recv.recv_timeout(Duration::from_millis($duration)).is_err() {
-        eprintln!("Command timeout. The proxy didn't send answer");
+        bail!("Command timeout. The proxy didn't send an answer");
```

Comment:
You change the behavior of the command here, it will crash with the new implementation

Reply:
I thought it was supposed to crash if the timeout is reached.

Reply:
Okay I replaced it with `eprintln!()` again.